docs/contributions.rst

```diff
@@ -63,17 +65,19 @@ Testing
 
 Add test cases, and run both existing cases and your new cases.
 
-If you add a new parser or modify existing parser, add new test cases
-to "Units". If you modify main part, add new test cases to
-"Tmain". The way to write and run test cases are written in "Testing
-ctags" of this document. Other than tmain test harness, you can
-specify VG=1 for running test cases under valgrind memory debugger.
+If you add a new parser or modify an existing parser, add new test
+cases to "Units". If you modify the core, add new test cases to
+"Tmain". The way to write and run test cases is described in "Testing
+ctags" of this document.
+
+You can specify VG=1 for running test cases under the Valgrind memory
+debugger.
```

Comment:
OK, sorry I'll restore the warning.  I just saw it was included under the tmain: target in Makefile and assumed it was now supported.